src/pages/SuperheroIndex.js

```diff
@@ -1,10 +1,31 @@
 import React, { Component } from 'react'
+import { Card, CardTitle, Col, CardImg, } from 'reactstrap'
+import { NavLink } from 'react-router-dom'
 
 class SuperheroIndex extends Component {
   render() {
     return (
       <>
-        <p>Index</p>
+         <h2>Supers Looking for Love!</h2>
+         <Col sm="6">
+        { this.props.superheroes.map((superhero, index) => {
+        return (
+        <Card body key={ index }>
```

Comment:
Card also needs to be indented correctly within <Col> so that the opening and closing tags are aligned